invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/errors/FormFeedback.js

```diff
@@ -217,7 +217,7 @@ class DisconnectedFormFeedback extends Component {
     const { icon, type } = feedbackConfig[feedback] || {};
 
     return (
-      <Message visible {...{ [type]: true }} className="flashed top attached">
+      <Message visible {...{ [type]: true }} className="flashed top attached" id={type + "-feedback-div"}>
```

Comment:
could we have a comment around what an expected value for `type` would be, and thus an expected resulting value for the ID?